### DISCUSSION TITLE
Client Initiated Backchannel Authentication (CIBA) equivalent to DeviceAuthorizationGrant API on Client

### DISCUSSION THREAD

**Discussion by FinHorsley** [panva/openid-client#469] [Q&A] [answered]
Hi all, new to the project so apologies if i've missed something obvious

Does the `Client` expose an api for CIBA? something equivalent to the `deviceAuthorization` method? e.g.

```ts
import { Issuer } from 'openid-client';

const issuer = await Issuer.discover(...);

const client = new issuer.Client({...});

const handle = await client.backChannelAuthorization(); //example

const tokenSet = await handle.poll();
```

**Comment by panva** [comment]
At the moment this library does not support CIBA, or FAPI-CIBA features.

**Comment by panva** [comment, marked as answer]
https://github.com/panva/openid-client/releases/tag/v6.2.0 added support for CIBA
